### DISCUSSION TITLE
Is the "Ignore this issue" feature supposed to persist between saves?

### DISCUSSION THREAD

**Discussion by JohnVeness** [openstreetmap/iD#9042] [Q&A] [answered]
I've been using the validator (Issues tab), and sometimes click on "Ignore this issue" to reduce clutter. However, after I save my changes, the ignored issues will reappear. I would rather they continued to be ignored after saving!

Before I report this as a bug or feature request, can I check first whether the current behaviour (resetting the ignored issues after a save) is as intended?

**Comment by bhousel** [comment, marked as answer]
The current behavior is intended.. After a save, all the data in iD is thrown away and refreshed from OSM's server.  So there can be completely different features, different node/way/relation ids, different issues, etc.  This refresh is pretty important for how the validator manages its internal state (partly to know which issues the user gets credit for fixing or causing).

**Reply by JohnVeness** [reply]
Thanks, I can see why you do that. A shame, but no worries!

I guess that also answers my issue #9016...
